library/s3/index.js

```diff
@@ -54,17 +54,19 @@ function s3Wrapper({ accessKeyId, secretAccessKey, region }) {
     } else {
       const excludeFiles = options.exclude || [];
       const filePathsArr = getFilePathsFromDirectory(sourceDirectory, excludeFiles);
-      for (const filePath of filePathsArr) {
-        // this allows the use of directory or directory/
-        const slash = s3Location[s3Location.length - 1] === '/' ? '' : '/';
-        const fullS3FilePath = `${s3Location}${slash}${filePath}`;
-        const s3UploadOptions = getS3UploadParameters({
-          bucket, key: fullS3FilePath, filePathToUplodad: filePath, options,
-        });
-        const waitInterval = options.throttleInterval || 100;
-        await new Promise((res) => { return setTimeout(res, waitInterval); });
-        await s3.putObject(s3UploadOptions).promise();
-        debugLog(`${filePath} to ${fullS3FilePath}`);
+      if (filePathsArr.length > 0) {
+        for (const filePath of filePathsArr) {
```

Comment:
I wouldn't think that this would break if there are no `filePath`'s in `filePathsArr`. What error were you seeing?